modern/src/attributes/positionAttributes.js

```diff
@@ -12,5 +12,10 @@ export default {
   'ignition': {
     name: t('positionIgnition'),
     type: 'boolean',
-  },  
+  },
+  'serviceOdometer': {
```

Comment:
This is rarely used attribute. I would recommend to use "odometer" instead.

Reply:
done